lib/Runtime/ByteCode/OpCodes.h

```diff
@@ -318,7 +318,7 @@ MACRO_EXTEND_WMS(       InitClassMemberSetComputedName,ElementI,    OpSideEffect
 MACRO_EXTEND_WMS(       InitClassMemberGetComputedName,ElementI,    OpSideEffect|OpHasImplicitCall|OpPostOpDbgBailOut)                  // Class member in get syntax with computed property name
 MACRO_EXTEND_WMS(       BrOnClassConstructor,       BrReg1,         None)               // Branch if argument is a class constructor
 
-MACRO_BACKEND_ONLY(     ArgIn_A,                    Empty,          OpSideEffect)       // Copy from "in slot" to "local slot", unchecked
```

Comment:
@ThomsonTan : Yes. It is safe to remove OpSideEffect for ArgIn. We talked about it during design to make sure that it really can't have any side effect. The reason for removing this attribute is to enable Dead store for ArgIn insturctions (where some formals that are not being used will be dead stored.)